pyxus/history.md

```diff
@@ -0,0 +1,66 @@
+# Unreleased
+- support deprecated flag in list_by_full_path and list_by_full_sub_path with False by default
```

Comment:
The history document is great! 
To be discussed: maybe, we could introduce a subsection with API changes such as the False by default

Reply:
I agree. I was not decided on how to do it but will put a subsection in v0.3.0 section mentioning API changes. 
We will see in the future if more formalism is needed